src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,32 @@
+@use '../_variables/variables' as *;
+
+.catalog {
+  margin: 0 auto;
+  padding: 50px 40px;
+  display: grid;
+
+  --column-count: 1;
+
+  grid-template-columns: repeat(var(--column-count), 200px);
+  place-content: center center;
+  justify-items: center;
+  gap: 46px 48px;
+}
+
+@media (min-width: $pov-phone) {
```

Comment:
The variable `$pov-phone` is used here, but it is not defined in this file. Ensure that it is imported correctly from the variables file or defined within this file.